src/integrators/gpt/gpt.cpp

```diff
@@ -1188,6 +1188,10 @@ class GradientPathTracer {
 GradientPathIntegrator::GradientPathIntegrator(const Properties &props)
 	: MonteCarloIntegrator(props)
 {
+	m_config.m_maxDepth = props.getInteger("maxDepth", -1);
+	m_config.m_minDepth = props.getInteger("minDepth", -1);
+	m_config.m_rrDepth = props.getInteger("rrDepth", 5);
+	m_config.m_strictNormals = props.getBoolean("strictNormals", false);

```

Comment:
I still think it's more clear if the config serializes all of its contents though, despite some of it being handled by the superclass